### ISSUE TITLE
No data in Domoticz

### ISSUE THREAD

**Issue by noppie** [Woutrrr/Omnik-Data-Logger#23]
Hi Woutrrr,

I followed your instructions but I can't manage to get any data in Domoticz.
The logfiles are also empty, while the config file says logging enabled.
Used the correct IDX for the virtual sensors in Domoticz, but no data...
Any suggestions?

**Comment by ikookmaar** [comment]
I'd like to have these instructions too, please.
The additions for exporting data to Domoticz described in https://www.domoticz.com/wiki/Omnik_Solar_Inverter won't work anymore, as config.cfg and OmnikExport.py have changed a lot since that version (september 2015), mainly through the use of "plugins". Maybe it's time for a Domoticz plugin?

**Comment by motor4all** [comment]
Is the problem the config.cfg? Some syntaxes have change since before. If you want I can post an example

**Comment by hans4307** [comment]
I've edited the Domoticz wiki. 
Getting your omnik data in Domoticz should be a piece of cake again.

**Comment by noppie** [comment]
Thank you.
Followed your instructions but now i’m getting the next error:

File "DomoticzOutput.py", line 61
SyntaxError: Non-ASCII character '\xc2' in file DomoticzOutput.py on line 61, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
root@raspberrypi:/home/pi/Omnik-Data-Logger/Omnik-Data-Logger# python DomoticzOutput.py 


Any idea?

**Comment by hans4307** [comment]
In the DomoticzOutput.py file, delete the degrees symbol in the comment at
line 61.
Line 61 is where the inverter temperature is send to domoticz.
I didn't test it yet, but hopefully this is the solution for your problem.
Please let me know if the problem isn't solved.

Hans.

Op 20 feb. 2017 6:59 p.m. schreef "noppie" <notifications@github.com>:

**Comment by noppie** [comment]
Yes I already figured that out, now i’m looking for a virtual sensor for my solar panles, but I don’t know wich one

**Comment by hans4307** [comment]
I'm using the virtual sensor 'Electric (Instant + Counter)'.
This one is standard for recording the amount of electricity used, but when you click on the edit-button of the sensor in the utility tab, you can change the type from usage to return.

**Comment by noppie** [comment]
Thanks,
How to change the icon….. now it looks like this:

**Comment by noppie** [comment]
Found it!  Just change from consumption to delivery…..

Everything works now!
